stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py

```diff
@@ -26,18 +26,20 @@ class FilterConformanceClasses(str, Enum):
         "http://www.opengis.net/spec/ogcapi-features-3/1.0/conf/features-filter"
     )
     ITEM_SEARCH_FILTER = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter"
-    CQL_TEXT = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:cql-text"
-    CQL_JSON = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:cql-json"
-    BASIC_CQL = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:basic-cql"
-    BASIC_SPATIAL_OPERATORS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:basic-spatial-operators"
-    BASIC_TEMPORAL_OPERATORS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:basic-temporal-operators"
-    ENHANCED_COMPARISON_OPERATORS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:enhanced-comparison-operators"
-    ENHANCED_SPATIAL_OPERATORS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:enhanced-spatial-operators"
-    ENHANCED_TEMPORAL_OPERATORS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:enhanced-temporal-operators"
-    FUNCTIONS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:functions"
-    ARITHMETIC = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:arithmetic"
-    ARRAYS = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:arrays"
-    QUERYABLE_SECOND_OPERAND = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:queryable-second-operand"
+    CQL_TEXT = "http://www.opengis.net/spec/cql2/1.0/conf/cql2-text"
+    CQL_JSON = "http://www.opengis.net/spec/cql2/1.0/conf/cql2-json"
+    BASIC_CQL = "http://www.opengis.net/spec/cql2/1.0/conf/basic-cql2"
+    BASIC_SPATIAL_OPERATORS = (
+        "http://www.opengis.net/spec/cql2/1.0/conf/basic-spatial-operators"
+    )
+    TEMPORAL_OPERATORS = " http://www.opengis.net/spec/cql2/1.0/conf/temporal-operators"
+    ADVANCED_COMPARISON_OPERATORS = (
+        "http://www.opengis.net/spec/cql2/1.0/conf/advanced-comparison-operators"
+    )
+    SPATIAL_OPERATORS = "http://www.opengis.net/spec/cql2/1.0/conf/spatial-operators"
+    FUNCTIONS = "http://www.opengis.net/spec/cql2/1.0/conf/functions"
+    ARITHMETIC = "http://www.opengis.net/spec/cql2/1.0/conf/arithmetic"
+    ARRAYS = "http://www.opengis.net/spec/cql2/1.0/conf/array-operators"
 
```

Comment:
I don't know if these are supported, but these two are missing from this list. It might be a good idea to add them, even if they're unused, for completeness.

* Property-Property Comparisons: http://www.opengis.net/spec/cql2/1.0/conf/property-property
* Accent and Case-insensitive Comparison: http://www.opengis.net/spec/cql2/1.0/conf/accent-case-insensitive-comparison